src/Workspaces/CSharp/Portable/LanguageServices/CSharpSyntaxFactsService.cs

```diff
@@ -443,13 +412,13 @@ public bool IsIdentifierEscapeCharacter(char c)
         public bool IsValidIdentifier(string identifier)
         {
             var token = SyntaxFactory.ParseToken(identifier);
-            return IsIdentifier(token) && !token.ContainsDiagnostics && token.ToString().Length == identifier.Length;
+            return this.IsIdentifier(token) && !token.ContainsDiagnostics && token.ToString().Length == identifier.Length;
```

Comment:
`IsIdentifier` became an extension, and you are required to call an extension off of a non-implicit receiver in C#.